backend/api/tests/test_services.py

```
@@ -0,0 +1,394 @@
+import pytest
+from unittest.mock import patch, MagicMock
+from django.contrib.auth import get_user_model
+from django.utils import timezone
```

Ruff (0.8.2)

`django.utils.timezone` imported but unused

Remove unused import: `django.utils.timezone`

(F401)